### PR TITLE
Update to version 0.13.0.Final

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkrohling Could you review please?

File: hawkular-apm-server/Dockerfile [hunk 1]
```diff
@@ -10,13 +10,13 @@ LABEL io.k8s.description="Hawkular APM Server is an application monitoring solut
 # Labels consumed by Red Hat build service
 LABEL com.redhat.component="openshift-hawkular-apm-docker" \
       name="openshift3/hawkular-apm-server-rhel7" \
-      version="0.12.0.Final" \
+      version="0.13.0.Final" \
```
Comment (jpkrohling) [inline hawkular-apm-server/Dockerfile:13]: Nice, you remembered about this one :)

**Review by jpkrohling — APPROVED** [review]
LGTM!

**Comment by objectiser** [comment]
@jpkrohling Thanks!

**Pull request merged**